datafusion/functions-aggregate-common/src/aggregate.rs

```diff
@@ -171,6 +171,9 @@ pub trait AggregateExpr: Send + Sync + Debug + PartialEq<dyn Any> {
     fn get_minmax_desc(&self) -> Option<(Field, bool)> {
         None
     }
+
+    /// Get function's name, for example `count(x)` returns `count`
+    fn func_name(&self) -> &str;
```

Comment:
is there a reason this isn't `name()` ? `func_name` is fine, it just seems inconsistent with the rest of the code

Reply:
This is to identify function, there is `name()` already, but it includes arguments, which is not I want.
Alternative way is introduce `nullable()` for AggregateUDF, so we don't need name checking. Maybe I should done it before this PR.